fuzon-http/README.md

```diff
@@ -0,0 +1,23 @@
+# fuzon-http
+
+This is a web-server to deploy fuzon as a web-service.
+All ontologies are loaded once on server startup, and the indices are kept in memory.
+
+## Configuration
+
+The server takes a configuration file as input to determine what ontologies to load, and which collections to load them into. Collections are individual matchers which can be queried independently.
+
+## Installation
+
+```shell
+cd fuzon-http
+cargo build --release
+../target/release/fuzon-http --config config/example.json
+```
+
+## Usage
```

Comment:
**issue (non-blocking)**: this usage section is not very helpful to a newcomer. After running the installation above, the server runs but it's not clear how a user is supposed to interact with it. I suggest we go a bit more in detail because we're the ones setting (for example) the port in the code, but that's not clear here. 
Additionally, for someone less experienced in APIs, showing only the commands doesn't tell much. Some examples on what the output would look like and how the command `/top?collection={collection}&top={top}&query={query}`  works would be helpful. We can use the example config as an example 👍

Reply:
Indeed, I've added the host+port to the config file!

Reply:
added examples based on config file